service/controller/network/v1/resource/ipam/create.go

```diff
@@ -1,9 +1,62 @@
 package ipam
 
-import "context"
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/meta"
+
+	"github.com/giantswarm/cluster-operator/service/controller/network/v1/key"
+)
 
 // EnsureCreated takes care of cluster subnet allocation when
 // ClusterNetworkConfig object is created.
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	clusterNetworkCfg, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	accessor, err := meta.Accessor(obj)
```

Comment:
We do not need to parse the accessor here separately. We already have it with `clusterNetworkCfg`. Below we call `accessor.GetNamespace()` but we can also simply do `clusterNetworkCfg.GetNamespace()`.

Reply:
True. Will fix.